ui/admin/tests/acceptance/credential-store/read-test.js

```diff
@@ -54,53 +56,68 @@ module('Acceptance | credential-stores | read', function (hooks) {
     urls.staticCredentialStore = `${urls.credentialStores}/${instances.staticCredentialStore.id}`;
     urls.vaultCredentialStore = `${urls.credentialStores}/${instances.vaultCredentialStore.id}`;
     urls.unknownCredentialStore = `${urls.credentialStores}/foo`;
+    
     authenticateSession({});
+    await visit(urls.projectScope);
   });
 
-  test('visiting static credential store in /credential-stores', async function (assert) {
+  test('visiting static credential store', async function (assert) {
     assert.expect(2);
-    await visit(urls.credentialStores);
+    await click(`[href="${urls.credentialStores}"]`);
     await a11yAudit();
     assert.strictEqual(currentURL(), urls.credentialStores);
-    await visit(urls.staticCredentialStore);
+
+    await click(`[href="${urls.staticCredentialStore}"]`);
     await a11yAudit();
+
     assert.strictEqual(currentURL(), urls.staticCredentialStore);
   });
 
-  test('visiting vault credential store in /credential-stores', async function (assert) {
+  test('visiting vault credential store', async function (assert) {
     assert.expect(2);
-    await visit(urls.credentialStores);
-    await a11yAudit();
+    await click(`[href="${urls.credentialStores}"]`);
     assert.strictEqual(currentURL(), urls.credentialStores);
-    await visit(urls.vaultCredentialStore);
+
+    await click(`[href="${urls.vaultCredentialStore}"]`);
     await a11yAudit();
+
     assert.strictEqual(currentURL(), urls.vaultCredentialStore);
   });
 
   test('cannot navigate to a static credential store form without proper authorization', async function (assert) {
-    assert.expect(1);
+    assert.expect(2);
     instances.staticCredentialStore.authorized_actions =
       instances.staticCredentialStore.authorized_actions.filter(
         (item) => item !== 'read'
       );
-    await visit(urls.staticCredentialStore);
-    assert.notOk(find('main tbody .rose-table-header-cell:nth-child(1) a'));
+
+    await click(`[href="${urls.credentialStores}"]`);
+
+    assert.dom(`[href="${urls.staticCredentialStore}"]`).doesNotExist();
+    assert.dom(`[href="${urls.vaultCredentialStore}"]`).isVisible();
```

Comment:
I would recommend updating `.isVisible()` to `.exists()`. `isVisible()` might imply that it can be hidden which is not the case in the context of this test.
```suggestion
    assert.dom(`[href="${urls.vaultCredentialStore}"]`).exists();
```